scripts/template/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2016 - 3016 Infinite Red LLC.
```

Comment:
I hop Infinite Red is around for a millennia but we might want to change this 😆

Reply:
It's funny this is wrong everywhere lol